src/com/watabou/pixeldungeon/effects/Flare.java

```diff
@@ -87,13 +87,13 @@ public Flare( int nRays, float radius ) {
 		for (int i=0; i < nRays; i++) {
 			
 			float a = i * 3.1415926f * 2 / nRays;
-			v[0] = FloatMath.cos( a ) * radius;
-			v[1] = FloatMath.sin( a ) * radius;
+			v[0] = (float)Math.cos( a ) * radius;
+			v[1] = (float)Math.sin( a ) * radius;
 			vertices.put( v );
 			
 			a += 3.1415926f * 2 / nRays / 2;
```

Comment:
Our static analysis engine detects that your pull request includes a change that **uses the rough value of Math.PI**. According to the [best practice](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value), it's recommended to use the predefined library constant for code clarity and better precision.